src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -589,6 +593,9 @@ const DateTimePicker = ({
    * @returns {string} an interval string, starting point in time to now
    */
   const getIntervalValue = () => {
+    if (getTooltipValue) {
+      return getTooltipValue();
+    }
```

Comment:
`getIntervalValue` is called in more than one place, so we shouldn't call `getTooltipValue()` here.

```suggestion
```

Could we modify `tooltipText` below to use the callback, and fallback to getIntervalValue if it's not truthy? I believe we should also pass the `currentValue` to the callback.


```diff
           <TooltipDefinition
              align="start"
              direction="bottom"
-              tooltipText={getIntervalValue()}
+              tooltipText={getTooltipValue(currentValue) || getIntervalValue()}
              triggerClassName=""
            >
```

Reply:
Thanks. I'll make this change.